test/workbox-core/node/_private/test-quota.mjs

```diff
@@ -0,0 +1,58 @@
+import {expect} from 'chai';
+import sinon from 'sinon';
+
+import expectError from '../../../../infra/testing/expectError';
+import {executeCallbacks, registerCallback} from '../../../../packages/workbox-core/_private/quota.mjs';
+import {devOnly} from '../../../../infra/testing/env-it';
+
+describe(`workbox-core quota`, function() {
+  let sandbox;
+
+  before(function() {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(function() {
+    sandbox.restore();
+  });
+
+  devOnly.it(`registerCallback() should throw when passed a non-function`, async function() {
```

Comment:
Can you wrap all the tests related to `registerCallback()` in a describe block and then do a separate describe block for all the tests in `executeCallback()`.

Reply:
I don't know about that—there's only one test that's just `registerCallback()` (which only runs in `devOnly` mode), and all of the rest of them rely on a combination of the two methods. That seems like overkill?

Reply:
Done.